Release.toml

```diff
@@ -1,4 +1,4 @@
-version = "0.3.2"
+version = "0.3.3"
 
 [migrations]
 "(0.3.1, 0.3.2)" = ["migrate_v0.3.2_admin-container-v0-5-0.lz4"]
```

Comment:
Don't you need to specify migrations for the version as well?
```suggestion
[migrations]
"(0.3.1, 0.3.2)" = ["migrate_v0.3.2_admin-container-v0-5-0.lz4"]
"(0.3.2, 0.3.3)" = []
```

I remember running into `MissingMigration` from when I did not specify migrations for the release version during testing.
https://github.com/bottlerocket-os/bottlerocket/blob/9cfc83b11a735ab8a5b466763d77843527067e6b/sources/updater/update_metadata/src/lib.rs#L410-L415

Reply:
To clarify, without specifying the empty list. Running `Release.toml` through updata would generate a bad manifest that would cause `MissingMigrations` errors when `updog` tries to update to that version.

Reply:
It wasn't designed that way, so if you have to have a migrations entry for every version (even if there are none) then we should fix that bug...